export/planloader/planloader.h

```diff
@@ -0,0 +1,38 @@
+/* SPDX-License-Identifier: Apache-2.0 */
+
+#include <substrait/common/Io.h>
+
+extern "C" {
+
+using SerializedPlan = struct {
+  // If set, contains a serialized ::substrait::proto::Plan object.
```

Comment:
I've become used to `//` indicating "internal comment" and `///` indicating "formal documentation".  Do we want to stick to that convention?

Reply:
I'm not sure I want to formalize this interface.  If someone wants to use the C++ code they should be using the recently exposed substrait_io library.  This version is for wrapping purposes only so I'd like it to be more obscure.